packages/obligatron/src/obligations/tagging.ts

```diff
@@ -0,0 +1,56 @@
+import type { PrismaClient } from '@prisma/client';
+import type { ObligationResult } from '.';
+
+interface AwsResource {
+	arn: string;
+	service: string;
+	resource_type: string;
+	taggable: string;
+	tags?: Record<string, string> | null;
+}
+
+const hasRequiredTags = (tags?: Record<string, string> | null): boolean =>
+	typeof tags === 'object' &&
+	tags?.['Stack'] !== undefined &&
+	tags['Stage'] !== undefined &&
+	tags['App'] !== undefined &&
+	tags['gu:repo'] !== undefined;
+
+const isExemptResource = (resource: AwsResource): boolean => {
+	if (resource.resource_type === 'role') {
+		// AWS Creates roles for various services when onboarding them.
+		// Technically we can tag these but they wouldn't belong to a specific App, Stage, or Stack.
+		return resource.arn.includes('/aws-service-role/');
+	}
+
+	return false;
```

Comment:
```suggestion
	return ['ec2', 'iam'].contains(resource.service);
```

Exempt everything but EC2 and IAM for the purpose of keeping the Obligation simple for this quarter?